src/_nebari/stages/input_vars.py

```diff
@@ -300,6 +300,12 @@ def stage_07_kubernetes_services(stage_outputs, config):
                     "*/*": ["viewer"],
                 },
             },
+            "argo-workflows-jupyter-scheduler": {
```

Comment:
gives access to conda store to view only environments

Reply:
Isn't that what line 305 does?